napari/_vispy/layers/image.py

```diff
@@ -6,6 +6,7 @@
 import numpy as np
 from vispy.color import Colormap as VispyColormap
 from vispy.scene.node import Node
+from vispy.visuals import ImageVisual
```

Comment:
```suggestion
from napari._vispy.visuals.image import Image
```

This has a bugfix we're using internally (not sure if it's important here, but better be consistent between layers types). Below of course use `isinstance(node, Image)`.

Reply:
No. As far as I check the `LabelNode` does not have `napari._vispy.visuals.image.Image` in the list of superclasses.
https://github.com/napari/napari/blob/0ece6a517f0e61d129351f52673d3341ad07465b/napari/_vispy/visuals/labels.py#L31

Reply:
It probably should then... That class has a fix for some edge cases with boundaries failing... which should prbably be in vispy, but I don't know the state of that now. If `Image` needs it, almost for sure `Labels` does as well.

Reply:
Could we do this in separate PR?